common/lib/symbiosis/ssl.rb

```diff
@@ -1,5 +1,23 @@
+require 'symbiosis'
+
 module Symbiosis
+  # SSL knows about which SSL providers exist and provides SSL helper functions
   class SSL
     PROVIDERS ||= []
+
+    def self.call_hooks(domains_with_updates, event)
+      return if domains_with_updates.empty?
```

Comment:
I don't think this is required.  The scripts should cope with no domains (and this shouldn't be called if there are no domains).